src/components/molecules/event/eventResponses/EventResponses.tsx

```diff
@@ -155,6 +209,24 @@ const EventResponses: React.FC<{
       },
       header: 'Bekreftet',
     },
+    {
+      cell: (cellValues) => {
+        const { attended } = cellValues;
```

Comment:
This can be `const { attended, email } = cellValues`, and we can remove the next line